docs/reference/connector/apis/update-connector-filtering-api.asciidoc

```diff
@@ -6,19 +6,22 @@
 
 preview::[]
 
+Updates the draft `filtering` configuration of a connector and marks the draft validation state as `edited`. The draft filtering can be then activated after a successful validation of the draft by the {enterprise-search-ref}/connectors.html[Elastic connector service].
```

Comment:
```suggestion
Updates the draft `filtering` configuration of a connector and marks the draft validation state as `edited`. The draft filtering can be then activated after a successful validation of the draft by the Elastic connector service.
```

Think this link might be jarring here.

Reply:
Again "The filtering configuration can be activated once validated by the Elastic connector service." feels passive, what does the user concretely have to do to activate?